### PR TITLE
setup-and-mobile-version-skeleton

### PR DESCRIPTION
**Pull request by brainskev**
Used design template
imported recommended fonts
Added header
Added recommended text styling

### CODE REVIEW

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @brainskev, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

 ## Required Changes ♻️

- [ ] Please as required by the previous reviewers, you should make sure that your design stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
  - Kindly fix the white space to the right side of your website, you need to remove the width of your `section.Headline` class in your `CSS` file 🙃 🙃
  - Kindly make sure the space between your elements is the same as in the Figma design 😁 
  - The background image has a border-radius to the left, please kindly fix that as well 😁 😁

Your design

Screen Shot 2022-10-28 at 17 26 49

Figma design

Screen Shot 2022-10-28 at 17 27 18


- [ ] As required by the previous reviewers the hamburger should be inside the nav tag. The nav tag is for semantics reasons, please kindly put it inside the `nav` tag 😁 😁 

Screen Shot 2022-10-28 at 17 31 28

- [ ] Please don't open a new Pull Request when you will ask for the next review. It's better to submit the same Pull Request, which allows code reviewers to proceed to the review without any confusion, and also code reviewers can see directly what changes the previous reviewer requested from you 🙃 🙃 

## Optional suggestions

- [OPTIONAL] I suggest you add the Getting started information in your README file, explain to users how they can get your project locally, how they can open it locally, things that they need in their device,... 🙃 😉
- [OPTIONAL] The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Please remember this in your next commits 😁 😉 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -11,14 +11,15 @@
...
                 <a href=""> My Logo</a>
                 <div class="humberger">
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                    <span class="bar"></span>
+                    <nav class="bar"></nav>
+                    <nav class="bar"></nav>
+                    <nav class="bar"></nav>
+                    
                 </div>
```
Comment (pinkmoon25) [inline index.html]: [OPTIONAL] ad requested by the previous reviewer this section shoul be like this :point_down: , as this is semantically more appropriate
```
<nav class="humberger">
            <div class="bar"></nav>
            <div class="bar"></nav>
             <div class="bar"></nav>
</nav>
```

File: index.html [hunk 2]
```diff
@@ -27,39 +28,39 @@
...
-                    <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667
-                    2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#505F79
-"/>
+                <div class="headline text">
+                   <h1>I’m kelvin <br> Glad to see you!</h1>
+                   <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="social-media">
+                    <p id="two">LET'S CONNECT</p>
```
Comment (pinkmoon25) [inline index.html:37]: [OPTIONAL] As this is a call to action putting it inside a link <a> or button tag would be more appropriate, what do you think? :thinking:

File: index.html [hunk 3]
```diff
@@ -27,39 +28,39 @@
...
                     </svg></a>
```
Comment (pinkmoon25) [inline index.html]: Each element inside a `<ul>` tag id generally `<li>` tag, you have placed the social icons in a` <a>` tag which is correct :+1: , but i would suggest please add each of these `<a>` tags inside a `<li>` tag

**Review by pinkmoon25 — APPROVED** [review]
Hi @brainskev , :wave:  

Great job implementing all the changes :clap: 
Your projct is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 No linters errors :heavy_check_mark: 
 Descriptive PR :heavy_check_mark: 
 Correct Github flow  :heavy_check_mark: 
 Used flexbox for layout  :heavy_check_mark: 

## Optional suggestions
_**I have added some optional changes in order to improve our site, please see them below**_ :pray: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**